### PR TITLE
Add notifications with rewards ff

### PR DESCRIPTION
**Pull request by jowlee**
### Description
Add with rewards feature flag for client to get notifications with rewards

Related to PR: https://github.com/AudiusProject/audius-protocol/pull/2321
Closes AUD-1272

### How Has This Been Tested?
Ran locally against local identity with staging db seed

### CODE REVIEW

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/jowlee-with-rewards-ff

File: src/common/services/remote-config/feature-flags.ts [hunk 1]
```diff
@@ -37,7 +38,8 @@ export const flagDefaults: { [key in FeatureFlags]: boolean } = {
   [FeatureFlags.ARTIST_RECOMMENDATIONS_ENABLED]: false,
   [FeatureFlags.SURFACE_AUDIO_ENABLED]: false,
   [FeatureFlags.PREFER_HIGHER_PATCH_FOR_PRIMARY]: true,
-  [FeatureFlags.PREFER_HIGHER_PATCH_FOR_SECONDARIES]: true
+  [FeatureFlags.PREFER_HIGHER_PATCH_FOR_SECONDARIES]: true,
+  [FeatureFlags.REWARDS_NOTIFICATIONS_ENABLED]: true
```
Comment (sddioulde) [inline src/common/services/remote-config/feature-flags.ts:42]: do we want a default of false instead?

**Review by sddioulde — APPROVED** [review]

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/jowlee-with-rewards-ff

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/jowlee-with-rewards-ff

**Pull request merged**